### PR TITLE
Add zh_TW Traditional Chinese translation for Android client

### PR DESCRIPTION
**Pull request by PeterDaveHello**
:tada:

### CODE REVIEW

File: client/Android/Studio/freeRDPCore/src/main/res/values-zh_rTW/strings.xml [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+        <item>800x600</item>
+        <item>1024x768</item>
+        <item>1280x1024</item>
+        <item>1440x900</item>
+        <item>1920x1080</item>
+        <item>1920x1200</item>
+    </string-array>
+    <!-- This array is used as is in code,
+	do not translate! -->
+    <string-array name="resolutions_values_array" translatable="false">
```
Comment (akallabeth) [inline client/Android/Studio/freeRDPCore/src/main/res/values-zh_rTW/strings.xml:90]: please remove all elements that have `translatable="false"` set.
they are not used in UI elements but as identifiers in code.

**Review by akallabeth — CHANGES_REQUESTED** [review]

File: client/Android/Studio/freeRDPCore/src/main/res/values-zh_rTW/strings.xml (hunk 1 above)
Comment (PeterDaveHello) [reply client/Android/Studio/freeRDPCore/src/main/res/values-zh_rTW/strings.xml:90]: No problem, just I saw that they are inside some locales, so didn't notice that they remove be removed :sweat_smile:

**Review by akallabeth — APPROVED** [review]
LGTM

**Comment by akallabeth** [comment]
@freerdp-bot test

**Review by akallabeth — CHANGES_REQUESTED** [review]
the folder should be named `values-zh-rTW` and not `values-zh_rTW`.
the build is failing because of this.

**Comment by PeterDaveHello** [comment]
Oops, updated now.

**Comment by akallabeth** [comment]
@freerdp-bot test

**Review by akallabeth — APPROVED** [review]
LGTM

**Pull request merged**